spring-cloud-loadbalancer/src/test/java/org/springframework/cloud/loadbalancer/core/LoadBalancerTests.java

```diff
@@ -116,17 +116,17 @@ public void emptyHosts() {
 	@Test
 	public void staticConfigurationWorks() {
```

Comment:
Can we add a new test for the list supplier so we can keep the existing test until it goes away.

Reply:
We could - I just wanted to remove deprecated elements wherever possible.

Reply:
Will add it back.